api/src/main/java/com/velocitypowered/api/event/command/ProxyCommandExecuteEvent.java

```diff
@@ -8,17 +8,21 @@
 package com.velocitypowered.api.event.command;
 
 import com.google.common.base.Preconditions;
+import com.velocitypowered.api.command.CommandResult;
 import com.velocitypowered.api.command.CommandSource;
 
 /**
  * This event is fired when velocity executed a command. This event is called after the event
  * is handled. It's not fired for forwarded commands or commands with syntax errors.
+ *
+ * <p>Commands can be cancelled or forced to backend servers in {@link CommandExecuteEvent}
+ * this event will not be caled in those cases.</p>
  */
-public final class PostCommandExecuteEvent {
+public final class ProxyCommandExecuteEvent {
```

Comment:
One name idea: `PostCommandInvocationEvent`. It's long, but it follows the same naming as the already present [`CommandInvocation`](https://github.com/PaperMC/Velocity/blob/36cc2bde23e7d50545c04b4060a52f3e6b461368/api/src/main/java/com/velocitypowered/api/command/CommandInvocation.java#L15) class, which "provides information related to the **possible execution** of a command".